_faq/about/03-costs.md

```diff
@@ -6,8 +6,12 @@ tags:
 ---
 ## Registration
 
-Registration for each event will be via a simple Google Form owned by Soc-RSE. All data will be deleted from the form after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors. 
+Registration for each event will be via dedicated registration link on its webpage.
```

Comment:
It does not become clear what *on its webpage* means. Do you refer to the page advertising the contribution on the website or do you mean the event pages in indico?

Reply:
it's the webpage of the event at sorse.github.io, http://www.philipp-s-sommer.de/sorse.github.io/programme/software_demos/event-001/ for instance. any ideas how to rephrase it to make it more clear @eileen-kuehn?

Reply:
You might simply say the following.

```suggestion
Registration for each event will be via dedicated registration in Indico.
```

To ensure that the following sentence properly describes the data you could then go on with "All registration data will be deleted [...]".

Reply:
Sounds good. But I think we should still tell them _where_ to find this registration link. Maybe something like
```suggestion
Registration for each event will be via dedicated registration in Indico.
Simply select the event from the [events calendar](){: .missing} and click the 
_Register_ button on its webpage.
```

Reply:
Great! I like your solution. 
I only compared to the previous version where we also didn't tell them where to go O:)

What I just realised. The first sentence somehow reads strange (maybe just for me as a German?). We could further improve by adding *available*.

```suggestion
Registration for each event will be available via dedicated registration in Indico.
Simply select the event from the [events calendar](){: .missing} and click the 
_Register_ button.
```

Reply:
Sounds good! What about

```suggestion
Registration for each event will be available via Indico. Select the event from the 
[events calendar](){: .missing} and click the _Register_ button on its webpage.
```

I think that's straight-forward to understand

Reply:
And another one skipping *on its webpage* :) I think click the button is sufficient.

```suggestion
Registration for each event will be available via Indico. Select the event from the 
[events calendar](){: .missing} and click the _Register_ button.
```